<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>180</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. X (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ZSteeringSpinBox">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>93</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>94</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ZSteering  (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="DistanceSkinLabel">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>130</y>
     <width>76</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>131</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Distance from
skin (mm) :</string>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="XMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>178</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>210</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. Y (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="YMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>208</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculatePlanningMask">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>340</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Calculate Fields</string>
   </property>
  </widget>
  <widget class="QWidget" name="AcField_plot1" native="true">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>30</y>
     <width>868</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>241</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. Z (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ZMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>239</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="RefocusingcheckBox">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>23</y>
     <width>27</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>22</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Refocusing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DistanceConeToFocusSpinBox">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>60</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>61</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance Cone to Focus (mm)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
